tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import functools
+import logging
+import os
+import os.path
+
+import pytest
+
+from pytest_bdd import given
+from pytest_bdd import parsers
+from pytest_bdd import then
+from pytest_bdd import when
+
+from utils.helper import run_ansible_playbook
+
+
+
+def pytest_addoption(parser):
```

Comment:
Let's stick to one way of passing this on. `ANSIBLE_INVENTORY` in the environment makes most sense (it's an Ansible standard), so let's stick to it and not add code to support another mechanism we don't use anyway?